packages/front-end/components/Features/SDKEndpoints.tsx

```diff
@@ -93,71 +92,58 @@ const SDKEndpoints: FC<{
               const endpoint = getSDKEndpoint(key.key, selectedProject);
 
               const envExists = environments?.some((e) => e.id === env);
+
               return (
                 <tr key={key.key}>
-                  <td>{key.description}</td>
-                  <td>
+                  <td className="d-flex flex-column">
                     <Tooltip
                       body={
                         envExists
                           ? ""
                           : "This environment no longer exists. This SDK endpoint will continue working, but will no longer be updated."
                       }
                     >
-                      {env}{" "}
+                      <strong className="mr-1">{env}</strong>
                       {!envExists && (
                         <FaExclamationTriangle className="text-danger" />
                       )}
                     </Tooltip>
+                    <span style={{ fontSize: "87.5%", fontStyle: "italic" }}>
+                      {key.description}
+                    </span>
                   </td>
                   <td>
-                    <code>{endpoint}</code>{" "}
-                    <FaCopy
-                      className="cursor-pointer"
-                      title="Copy to Clipboard"
-                      onClick={(e) => {
-                        e.preventDefault();
-                        navigator.clipboard
-                          .writeText(endpoint)
-                          .then(() => {
-                            console.log("Copied!");
-                          })
-                          .catch((e) => {
-                            console.error(e);
-                          });
-                      }}
-                    />
+                    <ClickToCopy valueToCopy={endpoint}>
+                      <span style={{ overflowWrap: "anywhere" }}>
```

Comment:
I think we want `wordBreak: "break-all"` instead.  Looks like `overflow-wrap: anywhere` is not supported on Safari.  https://caniuse.com/mdn-css_properties_overflow-wrap_anywhere